From 8f6ab2328c7f66848489b376d63aa2b3bda60aff Mon Sep 17 00:00:00 2001
From: ningl <ningl@codeaurora.org>
Date: Wed, 28 Sep 2016 15:53:38 +0800
Subject: [PATCH 09/32] Launcher3: No Calendarwidget 4x4 widget displayed

There is not enough space to place the Calendarwidget when in the
portrait mode for the mobile phone of 480*854.

Config the value of dynamic_grid_edge_margin a little smaller for the
mobile phone of 480*854.

Change-Id: I7be230b6539e47be4bd967c08eaf6f00aa550f75
CRs-Fixed: 1066228
---
 res/values-sw320dp/dimens.xml | 35 +++++++++++++++++++++++++++++++++++
 res/values-sw340dp/dimens.xml |  2 ++
 2 files changed, 37 insertions(+)
 create mode 100644 res/values-sw320dp/dimens.xml

diff --git a/res/values-sw320dp/dimens.xml b/res/values-sw320dp/dimens.xml
new file mode 100644
index 000000000..2825ca9ae
--- /dev/null
+++ b/res/values-sw320dp/dimens.xml
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+    Copyright (c) 2016, The Linux Foundation. All rights reserved.
+
+    Redistribution and use in source and binary forms, with or without
+    modification, are permitted provided that the following conditions are
+    met:
+    * Redistributions of source code must retain the above copyright
+    notice, this list of conditions and the following disclaimer.
+    * Redistributions in binary form must reproduce the above
+    copyright notice, this list of conditions and the following
+    disclaimer in the documentation and/or other materials provided
+    with the distribution.
+    * Neither the name of The Linux Foundation nor the names of its
+    contributors may be used to endorse or promote products derived
+    from this software without specific prior written permission.
+
+    THIS SOFTWARE IS PROVIDED "AS IS" AND ANY EXPRESS OR IMPLIED
+    WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF
+    MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT
+    ARE DISCLAIMED.  IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS
+    BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+    CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+    SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR
+    BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY,
+    WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE
+    OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN
+    IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+-->
+
+<resources>
+
+    <dimen name="dynamic_grid_edge_margin">4dp</dimen>
+
+</resources>
\ No newline at end of file
diff --git a/res/values-sw340dp/dimens.xml b/res/values-sw340dp/dimens.xml
index c9f2981f7..2c7ae8330 100644
--- a/res/values-sw340dp/dimens.xml
+++ b/res/values-sw340dp/dimens.xml
@@ -21,4 +21,6 @@
     <dimen name="drop_target_drag_padding">20dp</dimen>
     <dimen name="drop_target_text_size">16sp</dimen>
 
+    <dimen name="dynamic_grid_edge_margin">6dp</dimen>
+
 </resources>
\ No newline at end of file
-- 
2.17.1

